Sources/PrimerSDK/Classes/User Interface/OAuth/PrimerWebViewController.swift

```diff
@@ -56,9 +47,15 @@ internal class PrimerWebViewController: PrimerViewController, WKNavigationDelega
         webView.navigationDelegate = self // Control which sites can be visited
         view = webView
         if let url = url {
+            let frameworkVersion = Bundle.primerFramework.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
+            
             var request = URLRequest(url: url)
             request.timeoutInterval = 60
-            request.allHTTPHeaderFields = headerFields
+            request.allHTTPHeaderFields = [
```

Comment:
I feel that this logic + `frameworkVersion` could be extracted in another function

Reply:
Done!